### ISSUE TITLE
Relationship edge_labels default to related_to

### ISSUE THREAD

**Issue by isuen** [NCATS-Tangerine/ndex-beacon#23]
Right now edge_labels that do not exactly match a biolink category will default to related_to. It would be nice to have a heuristic to suss out somewhat more useful information

**Comment by RichardBruskiewich** [comment]
I agree. The ndex bio /predicates call is doing a very nice job of capturing the 'local' predicates now, so perhaps we can attempt a more sensible mapping to the Biolink predicate list (the "Slots" under https://biolink.github.io/biolink-model/docs/)
